megamek/src/megamek/common/weapons/ArtilleryWeaponIndirectFireHandler.java

```diff
@@ -435,7 +455,7 @@ else if ((null != bestSpotter) && !(this instanceof ArtilleryWeaponDirectFireHan
         } else {
             handleArtilleryDriftMarker(targetPos, finalPos, aaa,
                     gameManager.artilleryDamageArea(finalPos, aaa.getCoords(), atype,
-                            subjectId, ae, isFlak, height, mineClear, vPhaseReport,
+                            subjectId, ae, df, isFlak, height, vPhaseReport,
```

Comment:
I know you are not "fixing"it, but update the variables to use our current naming convention, aka: `ae`, `df` and `atype`. Its minor, won't cause any problems and will help in the long run.

Reply:
I'll do a POC, but if renaming them exceeds 100 lines changed I'm going to abort.

Reply:
Doing a variable rename, even if 100+ LOC, is still worth it long term. But this is your call as you're the one doing the work.